gramps/gen/utils/docgen/odstab.py

```diff
@@ -335,7 +335,8 @@ def open(self,filename):
 
         try:
             self.content_xml = tempfile.mktemp()
-            self.f = open(self.content_xml,"wb")
+            with open(self.content_xml,"wb") as self.f:
```

Comment:
Ok, then that makes sense. But it doesn't need to save the value in self.f then does it?